src/pages/robot/_refactor/account/hooks/useGetLog.ts

```diff
@@ -0,0 +1,79 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { GetTxsEventResponse } from 'cosmjs-types/cosmos/tx/v1beta1/service';
+import { CYBER } from 'src/utils/config';
+
+const LIMIT = 20;
+
+const request = async (address: string, offset: number, limit: number) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${CYBER.CYBER_NODE_URL_LCD}/cosmos/tx/v1beta1/txs?pagination.offset=${offset}&pagination.limit=${limit}&orderBy=ORDER_BY_DESC&events=cyberlink.particleFrom%3D%27QmbdH2WBamyKLPE5zu4mJ9v49qvY8BFfoumoVPMR5V4Rvx%27&events=cyberlink.neuron%3D%27${address}%27`,
```

Comment:
lets use this syntax
<img width="417" alt="image" src="https://github.com/cybercongress/cyb/assets/18665326/d6e4451d-9a31-4780-9890-dde400f02d7e">

Reply:
i think we should use soft3js for all requests